src/import/prepare.rs

```diff
@@ -185,6 +196,13 @@ impl<Client: ImportClient> PrepareRangeJob<Client> {
         }
         match self.split_region(&region) {
             Ok(new_region) => {
+                // We have to wait for few milliseconds because the PD may haven't received
+                // any heartbeat from the new split region, such that the PD cannot create
+                // scatter operator for the new split region because it doesn't have the
+                // meta data of the new split region
+                if self.cfg.wait_before_scatter_ms > 0 {
+                    thread::sleep(Duration::from_millis(self.cfg.wait_before_scatter_ms));
```

Comment:
Can we interact with PD and check if we can scatter the region?

Reply:
I think we can scatter region until we get new region info by `{$PD_HOST}/pd/api/v1/region/id/{id}`.

Reply:
@zhangjinpeng1987  What if we change this line to

```suggestion
                    while self.client.get_region_by_id(new_region.region.id)?.is_none() {
                        sleep(Duration::from_millis(10));
                    }
```

Reply:
Okay.